ClosedXML/Extensions.cs

```diff
@@ -132,6 +132,40 @@ public static String ToProper(this String value)
 
             return value.Substring(0, 1).ToUpper() + value.Substring(1);
         }
+
```

Comment:
I agree the 2 `StringExtensions` classes should be merged, but I still prefer to have them in a separate file. Don't you agree?

Reply:
Yeah, that makes sense.

Reply:
I've extracted all extension classes in separate files and put them in the Extensions folder.